patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -470,6 +470,15 @@
 +    }
 +
 +    /**
++     * Determines if the item has any additional overlays it should render.
++     * @param stack The current Item Stack
++     * @return True if it should fire {@link net.minecraftforge.client.event.RenderItemOverlayEvent} for this stack.
++     */
++    public boolean shouldRenderAdditionalOverlay(ItemStack stack) {
```

Comment:
I don't understand the point of having an item method *and* an event. It's usually one or the other.